### PR TITLE
checkValue script - for conditional tasks

### PR DESCRIPTION
**Pull request by gshriki**
Checking value of a field/label/context - helpful in conditional tasks. Similar to Print, with few diffs:
1. Better name for playbook authoring
2. Value argument is not mandatory (so if input is not set, we return false)

### CODE REVIEW

File: Scripts/script-checkValue.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+script: |-
+  //check if value is provided, otherwise return false
+  //needed for custom fields which if not set initially will be null
+  return (args.value) ? args.value : false;
+type: javascript
+tags:
+- Utility
+- Condition
+comment: 'Gets a value and return it. This is to be used in playbook conditional tasks
+  - get a value from incident field, label or context, and act accordingly. '
```
Comment (aviadl) [inline Scripts/script-checkValue.yml]: maybe add in the details using DT ?
If not you can merge

**Comment by aviadl** [comment]
array is indeed a problem, but it is different from exists in the sense that it returns the value, so it can be used in a switch case, and not only `yes`, `no`

**Comment by gshriki** [comment]
@slavikm - don't think this should not take arrays, as the intention is to use it in conditional tasks...

**Comment by slavikm** [comment]
@gshriki but what happens if you're passed an array? You should solve that case, no? Otherwise the behavior is a bit weird. On the other hand, maybe it's the normal behavior - just to return all the values and so the first value that matches a branch will be chosen...

**Comment by gshriki** [comment]
I'll run some tests to understand better the experience. Please do not merge yet.

**Comment by gshriki** [comment]
So if an array is returned, the first value will be the decision making value. Added clarification on that in the script description. Ready to be merged.

**Pull request merged**